templates/zerver/message_history.html

```diff
@@ -9,6 +9,6 @@ <h3 id="message-history-label">{{ _('Message edit history') }}</h3>
         <div class="controls" id="message-history"></div>
     </div>
     <div class="modal-footer">
-        <button class="btn btn-default" data-dismiss="modal" aria-hidden="true">{{ _("Close") }}</button>
+        <button class="btn btn-default" id="message-history-cancel" data-dismiss="modal" aria-hidden="true">{{ _("Cancel") }}</button>
```

Comment:
Let's keep the name close; we just changed it in ccd2e40c3ac1f68aa8c1d4a4e9b302f35fe58040 (I think maybe you resolved this merge conflict incorrectly?).